packages/components/psammead-navigation/src/DropdownNavigation/index.jsx

```diff
@@ -0,0 +1,232 @@
+  border: ${GEL_SPACING_HLF} solid ${C_WHITE};
+`;
+
+const CrossButton = styled.button`
+  background-color: ${C_EBON};
+  width: ${MENU_ICON_SIDE_LENGTH};
+  height: ${MENU_ICON_SIDE_LENGTH};
+  padding: 0;
+  margin: 0;
+  border: 0;
+
+  &:focus {
+    box-shadow: inset 0 0 0 ${GEL_SPACING_HLF} ${C_WHITE};
+    ::after {
+      ${iconBorder};
+    }
+  }
+`;
+
+export const DropdownUl = styled.ul.attrs({ role: 'list' })`
+  list-style-type: none;
+  margin: 0;
+  padding: 0 ${GEL_SPACING};
+  border-bottom: 0.125rem solid ${C_SHADOW};
+`;
+
+const DropdownLi = styled.li`
+  padding: 0.75rem 0;
+  border-bottom: 0.0625rem solid ${C_SHADOW};
+
+  &:last-child {
+    padding-bottom: ${GEL_SPACING_DBL};
+    border: 0;
+  }
+`;
+
+const StyledDropdownLink = styled.a`
+  ${({ script }) => script && getPica(script)};
+  ${({ service }) => service && getSansRegular(service)}
+  color: ${C_WHITE};
+  text-decoration: none;
+
+  &:focus {
+    text-decoration: underline;
+  }
+`;
+
+const StyledCurrentLink = styled.span`
+  ${({ dir }) => getStyles(dir)}
+`;
+
+export const DropdownNavigationLi = ({
+  children,
+  script,
+  currentPageText,
+  active,
+  service,
+  url,
+  dir,
+}) => (
+  <DropdownLi role="listitem">
+    <StyledDropdownLink script={script} service={service} href={url}>
+      {active && currentPageText ? (
+        <StyledCurrentLink
+          dir={dir}
+          // eslint-disable-next-line jsx-a11y/aria-role
+          role="text"
+        >
+          <VisuallyHiddenText>{currentPageText}</VisuallyHiddenText>
+          {children}
+        </StyledCurrentLink>
+      ) : (
+        children
+      )}
+    </StyledDropdownLink>
+  </DropdownLi>
+);
+
+DropdownNavigationLi.propTypes = {
+  children: string.isRequired,
+  url: string.isRequired,
+  script: shape(scriptPropType).isRequired,
+  service: string.isRequired,
+  active: bool,
+  currentPageText: string,
+  dir: oneOf(['ltr', 'rtl']),
+};
+
+DropdownNavigationLi.defaultProps = {
+  active: false,
+  currentPageText: null,
+  dir: 'ltr',
+};
+
+const dropdownProps = {
+  announcedText: string.isRequired,
+  children: node.isRequired,
+};
+
+export const CanonicalDropdown = ({ announcedText, onClose, children }) => (
+  <DropdownWrapper>
+    <CrossButton
+      aria-label={announcedText}
+      onClick={onClose}
+      // eslint-disable-next-line react/jsx-boolean-value
+      aria-expanded="true"
+    >
+      {navigationIcons.cross}
+    </CrossButton>
+    {children}
+  </DropdownWrapper>
+);
+
+CanonicalDropdown.propTypes = {
+  ...dropdownProps,
+  onClose: func.isRequired,
+};
+
+export const AmpDropdown = ({ announcedText, onClose, children }) => (
+  <DropdownWrapper>
+    <CrossButton
+      aria-label={announcedText}
+      on={onClose}
+      // eslint-disable-next-line react/jsx-boolean-value
+      aria-expanded="true"
+    >
+      {navigationIcons.cross}
+    </CrossButton>
+    <DropdownUl role="list">{children}</DropdownUl>
```

Comment:
The canonical dropdown just renders children, amp is putting the children within the Ul component - should we keep this consistent?

Reply:
Sorry, forgot to update the AMP version too